### ISSUE TITLE
<Epic> <CST A11y Audit Swarm> from <Benefits>

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#56264]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 
 
In recent discovery work conducted by @skylerschain , we noticed our [prior maintenance audit on the Claims Status Tool (CST) was incomplete](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447) and only began to scratch the service on potential a11y issues that might exist as documented in Skyler's [CST app map (Sketch)](https://www.sketch.com/s/d8594ef0-5efd-4a48-aa7a-baa3cb056482).

Given the potentially large scope of this work, I'm requesting an audit swarm (when specialists have bandwidth) to conduct a more holistic, joint a11y audit of CST.

Tasks
- [ ] @joshkimux to [identity screens, scope of audit, and login instructions](https://docs.google.com/spreadsheets/d/1jPbkog3fy-D1ltej_dcNUEwI4pvoentFkukn2VXDABQ/edit?usp=sharing)
- [ ] Plan and divide work among available specialists
- [ ] Produce a final report on a11y issues (using the above spreadsheet)
- [ ] Convert major issues into tickets attached to an epic

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [ ] No
  - [NA] 
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- [Past maintenance audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447)
- [Map of UI screens](https://www.sketch.com/s/d8594ef0-5efd-4a48-aa7a-baa3cb056482)
- [Swarm template for CST](https://docs.google.com/spreadsheets/d/1jPbkog3fy-D1ltej_dcNUEwI4pvoentFkukn2VXDABQ/edit?usp=sharing)

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date:
- Estimated staging review date:
- Content, Accessibility, and IA work needed by: TBD

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [x] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: Benefits Decision Reviews
-	OCTO-DE product owner: Matt Self
-	Product manager: Jacob Worrell
-	Designer: @skylerschain 
-	FE engineer: Kevin Duensing, Jerek Shoemaker, Sam Cormier
-	Product/team Slack channel: #benefits-decision-reviews

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by coforma-jamie** [comment]
@joshkimux, how high a priority is this ticket? Do you have a sense of when you'd like to start swarming on this?

**Comment by joshkimux** [comment]
@coforma-jamie thank you for the nudge, I should have updated this ticket sooner.
We've since half-completed our swarm audit, but came across blockers related to the testing environment (it was impossible to create reliable replicable steps).

We can close this ticket out, as we're refocusing our energy into simply redesigning the CST experience! We'll be working with y'all to focus on more structural/systemic issues with the IA/content in the near future.
